---
jackal_affordance/Object[] objects
jackal_affordance/Primitive[] primitives
bool success
